jetty-plus/pom.xml

```diff
@@ -43,7 +43,7 @@
         <extensions>true</extensions>
             <configuration>
               <instructions>
-                <Import-Package>javax.transaction*;version="[1.1,1.3)",*</Import-Package>
+                <Import-Package>javax.transaction*;version="[1.3)",*</Import-Package>
```

Comment:
You can't change this to require 1.3 as the minimum version unless you change the jetty-osgi/test-jetty-osgi tests to supply javax.transaction at version 1.3 too:  we have had to deliberately keep the version pegged back to 1.1 for the osgi tests because 1.2 and above have manifest entries with a lot of package imports that were impossible to satisfy, ie no jars available on maven central with good osgi manifests that exported those packages.

Reply:
Uhm. So for JPMS we need JTA 1.3, so this was to align the two.
Do you think we can be at 1.3 for JPMS and at 1.1 for OSGi?

Reply:
Looking at the dependency usage for javax.transaction(-api) I see ...
Jetty 9.4.x is at javax.transaction 1.2
Jetty 10.x is at javax.transaction 1.3